static/app/stores/organizationsStore.tsx

```diff
@@ -12,9 +12,9 @@ interface OrganizationsStoreDefinition extends StoreDefinition {
   getState(): Organization[];
   load(items: Organization[]): void;
   loaded: boolean;
-  onChangeSlug(prev: Organization, next: Organization): void;
+  onChangeSlug(prev: Organization, next: Partial<Organization>): void;
```

Comment:
This is also used as a partial if you look at what's being passed to the `changeSlug` action.